portfolio/src/main/webapp/script.js

```diff
@@ -122,17 +121,22 @@ function deleteTextFromDom(text) {
 
 function setCurrentPage() {
     const current = document.getElementById("current-page");
-    if (sessionStorage["current-page"]) {
-        current.innerText = sessionStorage["current-page"];
-    } else {
-        sessionStorage["current-page"] = 1;
-    }
+    sessionStorage["current-page"] = sessionStorage["current-page"] || 1;
+    current.innerText = sessionStorage["current-page"];
 }
 
+
 function changeButtonValUp() {
     const pagination = document.getElementById("pag");
     pagination.value = 1;
-    sessionStorage["current-page"]++;
+
+    const comments = document.getElementById("comment-section");
+    const numComments = comments.childElementCount;
+    console.log(numComments);
+    if (numComments == sessionStorage["num-comments"]) {
+        console.log("YOOOOOOO")
```

Comment:
lol I'd clean up the debug text if you don't need them anymore

Reply:
oops haha i just fixed it